### PR TITLE
Sprint 48: TLT-1870/dashboard card styling

### PR DESCRIPTION
**Pull request by elliottyates**

### CODE REVIEW

File: canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html [hunk 1]
```diff
@@ -21,67 +21,103 @@
...
+          <div class="card-header card-header_disabled">
+          </div>
+          <div class="card-content card-content_alert">
+            <h2 class="card-content-title ellipsis"
+                title="Import iSites Content">
+              Import iSites Content
+            </h2>
+            <p title="Import iSites Content">
+              Migrate content from a previous term's
+              Course iSite to this Canvas course
```
Comment (sapnamysore) [inline canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html:108]: Maybe it is in the design. But we want to display "Migrate content..." followed by the "This Canvas course is not associated   with any Course iSites." when there are no active migrations? (thought we will only have the latter)

Comment (sapnamysore) [inline canvas_course_admin_tools/templates/canvas_course_admin_tools/dashboard_course.html:108]: Just saw how it looks on Dev - Thanks!

**Comment by sapnamysore** [comment]
👍 
Not sure if I like the larger box (compared to the 2  boxes for MP and MS)  for iSites Migration when it is grey and has the notification in red. But that's more of a design thing and not directly related to the PR

**Comment by Vittorio2015** [comment]
Sapna, let's let it go for the sake of this user story and let's discuss design alternatives; I would be interested to know ideas and suggestions, especially since this design needs to be eventually scalable to who knows what additional tools, and with what conventions.

**Comment by sapnamysore** [comment]
Sounds good @Vittorio2015 ! Yes, had some ideas/suggestions to make, but it doesn't need to hold up this story. Thanks!

**Pull request merged**